modules/data-loading/pages/streaming-data-connector.adoc

```diff
@@ -0,0 +1,438 @@
+file.reader.settings.fs.gs.auth.service.account.email=gcsconnect@tigergraph-dev.iam.gserviceaccount.com
+file.reader.settings.fs.gs.auth.service.account.private.key.id=55c1d79a46c1f3f59ef72e0df53285a3eef8ec38
+file.reader.settings.fs.gs.auth.service.account.private.key="-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDSqbYRwD68FvA7\nLkC1HpjrJ9QIJ+iOyQPFeSoI+3pjmVTrX2B2aYIMByNubV6Js+n1x5ro/XW0nt3y\nk/BhdOTXj7JVRj6JMIb0yjsRQMi3J+3yOb2EFVHUDQ+4nmTuSJsdiOI1mh1pFN+Q\nXdvHP5hOwCaB4Pb/X7ya9YOokW3dVqbHtj/DO3l+rDhqEP0SH4+RInFbZon1AT3J\ncWDdMTsx4yW1PQNERzP/9M34du3ihWeT1xLLXquhMnFO+zECuPsoz1jFQrLCAFeX\nQSBx0/NgBCRqEsX4XESQ43bB4mD3D9AvfOc6IuYqKBcjG2HmmjOvidmnlRUgZJy/\n2rymIUXnAgMBAAECggEAR6itI0qmzGptG2R3ZGTdFZi9umyA4hkkrEaz8sxAbKLa\nzRnrgTwQnbDL76NKdkL6Ab39RuX45RDpZLvIGA6gTWc2/WTgnuAf+CLWht7np83w\nVeYoPkbWR/CNeXp/0MJn6VsHv74F5RnRlpUmzpcmYxtfvexdeK8DRB7hwzR9D73t\nCyad7O6NZabuOQBrTMgKL+So6gurVjW+KB8S9vgMvULLOmTZ1iUpRlh89cJ4/jRh\n4ltV+4EvBJvIlXD2GKMGRw8d/YPWmETO/dpz0aAs3sMgXdiFV3SAjAn8BgaYita8\nIAhLLOf/kFmFmmlM2k02iAZPIBjFvAs7ChGEHsXecQKBgQDq9AKPXaMOiy43EzHP\nU1cDKf8mwk9ELtfGPhySG9Z+zUclvoJnbq0XpP3cKJWgKILtcG+cUoYLSWrosE2i\n7W1976EwObNgg22CEICWclE0Mc2vMXkJT+ZUoqWWK5n83ZK+R05AqsBhWC9FLTQB\nBD5XlOnCUKQT3+3TRkNAB5I/6QKBgQDliKxQ2TR8OEkgII8ugyhVAfKamE2nVrED\nW0U/IbBQUaTk1niZbzROLRRfgqYqtLM0vEfMkKwiMDinmXsuurw2IOZaGbEyNFZT\nLpDXNjTTePJn192OT6wyRpDsuZNh+0uiXBFyJb8vRrMyWmtau9HXqD2kXOcKCesL\nVDfilsAFTwKBgCkKpsfUW39W4KPOPo0wyapL0745gw8t/5MplmQPaNCNmzgEp1La\nCnJu58llbX2klfpUAasU30Vpdbtf0K/9OXseONHrwmHBk4d8ynl9TqIHcR6BTdtK\nkbmHD9XDmAqLye5jFlBFg4V9mgRDeSoUS6+Q26SN4Zt3KlwVkfnFWM7BAoGBAJnr\nA3oXnQ1rhQXJL5qGEwamDrRCS1haVsskahQCmEPT69oUQ7zICHAf5JiDeMAMeltz\nokX4AaXPZj5lOmhEii9V8oIa1msPE5AmGrRmQhhI82xVIdnrbVItZcOIUd+Tbs2K\nJZzA2Spvo3yxi2nFptqRk/xi2/8sVXQ8XllQs6UbAoGAdqnrlEAIlCb5hdVNrLXT\nToqdq54G9g82L8/Y+WraqJSNOFKXCGQvC2N16ava4sZ65DCjT6FnCR/UhYS7Z6Vf\nR5EtMRYAyAcyn3g9tcfzINmEbpvwpHBqsr1xPcrfx/WRurIC6EBgLPgX+lALBI0G\n+Uu87tgHhcGFJfmQMQNeQWM=\n-----END PRIVATE KEY-----\n"
+file.reader.settings.client_email="gcsconnect@tigergraph-dev.iam.gserviceaccount.com"
+file.reader.settings.fs.gs.project.id="tigergraph-dev"
+file.reader.settings.fs.gs.auth.service.account.enable=true
+file.reader.settings.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem
+file.reader.settings.fs.AbstractFileSystem.gs.impl="com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS"
+
+mode=eof
+file.regexp=".*"
+file.recursive=true
+file.reader.type=text
+file.reader.batch.size=10000
+file.reader.text.eol="\\n"
+file.reader.text.header=true
+file.reader.text.archive.type=auto
+file.reader.text.archive.extensions.tar=tar
+file.reader.text.archive.extensions.zip=zip
+file.reader.text.archive.extensions.gzip=tar.gz,tgz
+
+[connector_person]
+name = fs-person-demo-104
+tasks.max=10
+topic=person-demo-104
+file.uris=gs://tg_csv/p.csv
+
+[connector_friend]
+name = fs-friend-demo-104
+tasks.max=10
+topic=friend-demo-104
+file.uris=gs://tg_csv/f.csv
+----
+
```

Comment:
It is better not to disclose our credentials. We can change them to made up entries or just '*' s.